app/views/batch_contexts/_new_bc_form.erb

```diff
@@ -43,11 +43,11 @@
                     as: :radio_buttons,
                     collection: [['Yes', true], ['No', false]] %>
                 <div data-ocr-target="runOcrImageNotes" hidden>
-                    <%= render 'alert', alert_type: 'warning', text: 'ocr_image' %>
+                    <%= render 'alert', alert_type: 'info', text: :ocr_image %>
```

Comment:
I think this should be `info` to be consistent with the document message below?

Reply:
I am not sure I understand. I think warning is consistent. The document has two messages. A warning that says if the doc doesn't have text don't ocr it and a note (info) that says autogenerating ocr doesn't fully meet accessiblity standards.